src/java.desktop/share/classes/javax/swing/plaf/synth/SynthDefaultLookup.java

```diff
@@ -35,6 +37,12 @@
  */
 class SynthDefaultLookup extends DefaultLookup {
     public Object getDefault(JComponent c, ComponentUI ui, String key) {
+        if (ui instanceof SynthOptionPaneUI) {
+            Object value = UIManager.get(key, c.getLocale());
+            if (value != null) {
+                return value;
+            }
```

Comment:
It should probably call `super.getDefault` rather than use `UIManager` directly.

Reply:
ok..